src/containers/unitsOfMeasuring/unitOfMeasuring-item.js

```diff
@@ -137,20 +137,12 @@ UnitOfMeasuringItemWrapper.defaultProps = {
   item: {},
   editedUnitOfMeasuring: null,
   allCategories: {},
-  editСancel: () => {},
-  save_unitOfMeasuring: () => {},
-  delete_unitOfMeasuring: () => {},
-  setUnitOfMeasuringEditedDispatch: () => {},
 };
 
 UnitOfMeasuringItemWrapper.propTypes = {
   item: PropTypes.object,
```

Comment:
You have propTypes on 96 line. Why you use two propTypes, you need only one. Transfer these propTypes in propTypes on 96 line.